spec/controllers/meeting_contents_controller_spec.rb

```diff
@@ -71,6 +71,25 @@
           let(:mail_count) { 3 }
         end
       end
+
+      context 'with an error during deliver' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.